addons/website_slides/controllers/main.py

```diff
@@ -64,10 +65,12 @@ def _fetch_slide(self, slide_id):
         return {'slide': slide}
 
     def _set_viewed_slide(self, slide, quiz_attempts_inc=False):
-        if request.env.user._is_public() or not slide.website_published or not slide.channel_id.is_member:
+        if not slide.channel_id.is_member:
             viewed_slides = request.session.setdefault('viewed_slides', set())
             if slide.id not in viewed_slides:
                 if tools.sql.increment_field_skiplock(slide, 'public_views'):
+                    # Increment total views immediately as compute is not triggered
+                    tools.sql.increment_field_skiplock(slide, 'total_views')
```

Comment:
A bit sad to make 2 different requests to update 2 fields on the same record.
But I guess the tooling does not allow to pass multiple fields, not a huge deal I guess.

If you have some extra time, we could make it so the tooling allows to take in multiple fields to increment (in a separate commit with tests).